e2e/README.md

```diff
@@ -14,10 +14,14 @@ curl -sSfL https://raw.githubusercontent.com/longhorn/longhorn/master/scripts/en
 
 ### Run the test
 
-1. Deploy all backupstore servers (including `NFS` server and `Minio` as s3 server) for test purposes.
+1. Deploy all backupstore servers (including `NFS` server and `Minio` as s3 server, `CIFS` and `Azurtize` server) for test purposes.
+
+   For Azurtize, there are some manual steps need to be done after manifest deployed(https://confluence.suse.com/display/LON/Setup+Azurite+Backupstore+For+Testing).
```

Comment:
@mantissahz @chriscchien Could this be resolve to eliminate the tedious manual steps?

Reply:
There is a dicsussion in https://github.com/longhorn/longhorn/issues/9340, currently the manual steps is the workaround to setup azurite in test environment. If the discussion having further conclusion, I will follow up to modify code to match the behavior. 
cc @mantissahz

Reply:
See if it's possible to resolve this.

Reply:
My only concern is this tedious workaround, which make it difficult for users to test Azurite on Longhorn. I'll wait and see if this part can be improved.